zeroex/orderwatch/order_watcher_test.go

```diff
@@ -128,17 +122,17 @@ func TestOrderWatcherUnfundedInsufficientERC20Balance(t *testing.T) {
 	meshDB, err := meshdb.New("/tmp/leveldb_testing/"+uuid.New().String(), ganacheAddresses)
 	require.NoError(t, err)
 
-	signedOrder := scenario.CreateZRXForWETHSignedTestOrder(t, ethClient, makerAddress, takerAddress, wethAmount, zrxAmount)
+	signedOrder := scenario.NewSignedTestOrder(t, orderopts.SetupMakerState(true))
```

Comment:
We could also avoid setting up this state.

Reply:
I'm not sure we can. If I understand the intent of this test correctly, we still want to set the allowance, which `NewSignedTestOrder` does. It requires less code to do it this way compared to setting up the allowance manually after calling `NewSignedTestOrder`.

Reply:
I meant that we could if we used the `init` trick. As you pointed out, we really can't, so this should be disregarded.